tests/truffle/HelloWorld/contracts/Migrations.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity >=0.4.21 <0.7.0;
```

Comment:
In contracts/HelloWorld.sol and test/HelloWorld.sol, we require Solidity = 0.5.8. Let's be consistent, either pick 0.5.8 or set these bounds on those other files.